shared-module/src/styles/componentStyles.ts

```diff
@@ -6,7 +6,6 @@ import { defaultContainerWidth } from "./constants"
 // in a page to be inside a container. Some elements need the entire width
 // of the page.
 export const normalWidthCenteredComponentStyles = css`
-  width: 100%;
```

Comment:
This causes issues with CMS justify center. i.e.
https://github.com/rage/secret-project-331/pull/294/files#diff-466d275516a10e001cafe8f4b266d13c5fbbddf8ff0fff84d85276357bf044b9R13

Removing this does not affect anything, as max-width > width and on exercises we unset the max-width thus they'll default to full width from parents.